python/gui/auto_generated/qgsadvanceddigitizingdockwidget.sip.in

```diff
@@ -356,9 +356,16 @@ Adds point to the CAD point list
 
     void removePreviousPoint();
 %Docstring
-Remove previous point in the CAD point list
+Removes previous point in the CAD point list
 
 .. versionadded:: 3.8
+%End
+
+    void updateCurrentPoint( const QgsPointXY &point );
+%Docstring
+Updates the current ``point`` in the CAD point list
+
+.. versionadded:: 3.30.2
```

Comment:
I'd skip this addition to simplify backporting and use the QgsPoint variant instead. Some QgsPointXY methods are marked deprecated anyway.